### PR TITLE
Sasha adds hometown city

### PR DESCRIPTION
**Pull request by sashayeu**
Burlington with a B

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Add your hometown
 |Caitlin   |Timmons        |         |
 |Vibha     |Gogu           |         |
 |Annah     |Mutaya         |         |
-|Sasha     |Yeutseyeva     |         |
+|Sasha     |Yeutseyeva     |  Burlington       |
```
Comment (amutaya) [inline README.md:33]: Add state and country

**Review by amutaya — CHANGES_REQUESTED** [review]
No shame in our game; please make the suggested changes

File: README.md (hunk 1 above)
Comment (sashayeu) [reply README.md]: Done!

**Comment by sashayeu** [comment]
@amutaya I'm done, can you approve?

**Review by amutaya — APPROVED** [review]
Looks good

**Pull request merged**